mkosi/__init__.py

```diff
@@ -3299,32 +3300,36 @@ def run_clean(args: Args, config: Config) -> None:
         remove_build_cache = args.force > 1
         remove_package_cache = args.force > 2
 
-    with complete_step("Removing output files…"):
-        if config.output_dir_or_cwd().exists():
-            rmtree(*(p for p in config.output_dir_or_cwd().iterdir() if p.name.startswith(config.output)))
+    if (outputs := list(config.output_dir_or_cwd().glob(f"{config.output}*"))):
+        with complete_step(f"Removing output files of {config.name()} image…"):
+            rmtree(*outputs)
 
     if remove_build_cache:
         if config.cache_dir:
-            with complete_step("Removing cache entries…"):
+            with complete_step(f"Removing cache entries of {config.name()} image…"):
                 rmtree(*(p for p in cache_tree_paths(config) if p.exists()))
 
         if config.build_dir and config.build_dir.exists() and any(config.build_dir.iterdir()):
```

Comment:
I think the any check can be removed since rmtree checks if it's paths argument is truthy.

Reply:
Nope, we don't want to log if there's no files in the build directory.

Reply:
Ok, though I don't think the log wouldn't be misplaced as long as the dir exists.